### ISSUE TITLE
lite is not a supported API version

### ISSUE THREAD

**Issue by bsiddiqui** [contextio/ContextIO-node#13]
errors when I try and do this
```js
var ctxioClient = new ContextIO.Client('lite', {
  key: 'mykey',
  secret: 'mysecret'
});
```

**Comment by Deesr** [comment]
+1

Same issue here. Would love to see lite support.

**Comment by hartwoolery** [comment]
+1

**Comment by sushithegreat** [comment]
I'll see what I can do about getting a Lite version out there for everyone.

**Comment by BigBlueHat** [comment]
So...this client's only for 2.0 then?

I guess just using a straight node.js HTTP client like `requests` is the best way to go in the interim?

Thanks!

**Comment by AaronLaBrie** [comment]
We've just released a new version that supports both 2.0 and Lite! 

It's a completely new library that will be supported going forward, check out the README for more info.
